pkg/registry/styles/projects.less

```diff
@@ -0,0 +1,92 @@
+
+@tag-border: @link-color;
+
+table.projects-listing {
+
+    #add-button-icon {
+        float: right;
+        color: #0099d3;
+        padding: 10px 0px 10px;
+    }
+
+    #add-project-icon {
+        float: right;
+        color: #0099d3;
+        padding: 10px 0px 10px;
+    }
+
+    #add-group-icon {
+        float: right;
+        color: #0099d3;
+        padding: 30px 0px 10px;
+    }
+
+    #add-user-icon {
+        float: right;
+        color: #0099d3;
+        padding: 30px 0px 10px;
+    }
+
+    tbody:hover {
```

Comment:
Why is this necessary? Shouldn't this be done by the listing stuff in cockpit.css?